crates/paralegal-flow/src/test_utils.rs

```diff
@@ -135,7 +135,12 @@ where
     S: AsRef<std::ffi::OsStr>,
 {
     paralegal_flow_command(dir)
-        .args(["--abort-after-analysis", "--dump", "serialized-flow-graph"])
+        .args([
+            "--abort-after-analysis",
+            "--dump",
+            "serialized-flow-graph",
+            "--eager-local-markers",
```

Comment:
Yeah I should really fix that.

Reply:
I'll wait for #89 to be merged and rm this.